publish/views/asset.py

```diff
@@ -46,7 +42,7 @@ class AssetViewSet(NestedViewSetMixin, DetailSerializerMixin, ReadOnlyModelViewS
 
     permission_classes = [IsAuthenticatedOrReadOnly]
     serializer_class = AssetSerializer
-    serializer_detail_class = AssetDetailSerializer
+    serializer_detail_class = AssetSerializer
```

Comment:
No need to include `serializer_detail_class` if it's the same as `serializer_class`.

Reply:
Additionally, we can remove the lines with `queryset_detail` and the `DetailSerializerMixin` itself, if we're not going to use it for now.

Reply:
> No need to include `serializer_detail_class` if it's the same as `serializer_class`.
> 
> [chibisov.github.io/drf-extensions/docs/#detailserializermixin](https://chibisov.github.io/drf-extensions/docs/#detailserializermixin)
> 
> > If you use DetailSerializerMixin and don't specify serializer_detail_class attribute, then serializer_class will be used.

If I make the change you suggested, I get the following error:
```
'AssetViewSet' should include a 'serializer_detail_class' attribute
```

Do you know a way around this?

Reply:
> Additionally, we can remove the lines with `queryset_detail` and the `DetailSerializerMixin` itself, if we're not going to use it for now.

Actually, if I make these changes as well, the errors go away. If we're not going to use the detail, I'm fine with committing that.